apps/library/Makefile

```diff
@@ -25,7 +25,7 @@ endif
 
 CFLAGS := $(BASE_CFLAGS)
 
-all: librvc out/bin/cr_api_test.elf out/bin/graphic_screen_api_test.elf out/bin/fpga_io_test.elf
+all: librvc out/bin/cr_api_test.elf out/bin/graphic_screen_api_test.elf
 
 librvc: out/lib/librvc.a obj/crt0.o
 
```

Comment:
General request - remind me to talk about his file. I would like to understand its purpose and how it works.